jetstream/cli.py

```diff
@@ -140,6 +144,7 @@ def execute(
 
                 config = spec.resolve(experiment)
                 self.analysis_class(self.project_id, self.dataset_id, config).run(date)
+                export_metadata(config, self.bucket)
```

Comment:
Alternatively, instead of running the export after the analysis, we could also have a separate step in the Argo workflow for exporting metadata. However that would require resolving the experiment config in each step.

Reply:
This should be inexpensive so I agree this is a good place for it